Formula/hbase.rb

```diff
@@ -163,6 +162,9 @@ def post_install
     system "#{bin}/start-hbase.sh"
     sleep 10
     begin
+      system "ls", "-al", testpath
+      system "ls", "-al", testpath/"pid"
+      system "ps", "-ef"
       assert_match "Zookeeper", pipe_output("nc 127.0.0.1 #{port} 2>&1", "stats")
```

Comment:
Test originally failed, but after adding debug commands it passed. May be a timing issue so can try increasing sleep on next run.